.ddev/config.yaml

```diff
@@ -64,7 +64,7 @@ composer_version: "2"
 # see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
 # For example Europe/Dublin or MST7MDT
 
-# composer_version: "2"
+# composer_version: ""
```

Comment:
Does it use composer v2 by default now?

Reply:
It depends on your setup. But as our packages require bootstrap v2 it is better to require it explicitely